pkg/helmcertifier/checks/checks_test.go

```diff
@@ -97,3 +97,33 @@ func TestHasReadme(t *testing.T) {
 		})
 	}
 }
+
+func TestContainsTest(t *testing.T) {
+	type testCase struct {
+		description string
+		uri         string
+	}
+
+	positiveTestCases := []testCase{
+		{description: "tarball contains at least one test", uri: "testchart-0.1.0.tgz"},
+	}
+
+	for _, tc := range positiveTestCases {
+		t.Run(tc.description, func(t *testing.T) {
+			r, err := ContainsTest(tc.uri)
+			require.NoError(t, err)
+			require.NotNil(t, r)
+			require.Equal(t, true, r.Ok)
+		})
+	}
+
+	negativeTestCases := []testCase{}
+
+	for _, tc := range negativeTestCases {
+		t.Run(tc.description, func(t *testing.T) {
+			r, err := ContainsTest(tc.uri)
+			require.Error(t, err)
```

Comment:
Negative test cases do not test for errors, but for reasons the check has failed; in other words, `result.Ok` is `false` and `result.Reason` contains the reason it is false (for example, stating no tests have been found in the chart).

Reply:
done